runtime/src/accounts_index.rs

```diff
@@ -319,23 +384,27 @@ impl<T: 'static + Clone> AccountsIndex<T> {
         (reclaims, slot_list.is_empty())
     }
 
+    pub fn min_ongoing_scan_root(&self) -> Option<Slot> {
+        self.ongoing_scan_roots
+            .read()
+            .unwrap()
+            .keys()
+            .cloned()
+            .next()
```

Comment:
Can it be just `.keys().next().clone()` ? You only need the first key right? not all keys cloned.

Reply:
yeah totally, fixed thanks!